code/obj/storage/gang_crate.dm

```diff
@@ -0,0 +1,1067 @@
+				spawn_item(C,I,/obj/item/reagent_containers/syringe/krokodil,off_y=3,rot=45,scale_x=0.7,scale_y=0.7)
+				spawn_item(C,I,/obj/item/reagent_containers/syringe/krokodil,rot=45,scale_x=0.7,scale_y=0.7)
+				spawn_item(C,I,/obj/item/reagent_containers/syringe/krokodil,off_y=-3,rot=45,scale_x=0.7,scale_y=0.7)
+
+		stims_syringe
+			weight=2
+			spawn_loot(var/C,var/I)
+				for(var/i=1 to 3)
+					var/obj/item/syringe = spawn_item(C,I,/obj/item/reagent_containers/syringe,off_y=6-3*i,rot=45,scale_x=0.7,scale_y=0.7)
+					var/stim = pick(strong_stims)
+					syringe.reagents.add_reagent(stim, 15)
+					syringe.name_suffix("([syringe.reagents.get_master_reagent_name()])")
+					syringe.UpdateName()
+		cigar
+			spawn_loot(var/C,var/I)
+				for(var/i=1 to 3)
+					var/obj/item/cig = spawn_item(C,I,/obj/item/clothing/mask/cigarette/cigar,off_y=3*(2-i))
+					cig.reagents.add_reagent("salicylic_acid", 5)
+					cig.reagents.add_reagent("CBD", 5)
+
+		goldcigar
+			spawn_loot(var/C,var/I)
+				for(var/i=1 to 3)
+					var/obj/item/cig = spawn_item(C,I,/obj/item/clothing/mask/cigarette/cigar/gold,off_y=3*(2-i))
+					cig.reagents.add_reagent("salicylic_acid", 5)
+					cig.reagents.add_reagent("omnizine", 5)
+
+		drug_injectors
+			spawn_loot(var/C,var/I)
+				spawn_item(C,I,/obj/item/reagent_containers/emergency_injector/random,off_y=4)
+				spawn_item(C,I,/obj/item/reagent_containers/emergency_injector/random,off_y=0)
+				spawn_item(C,I,/obj/item/reagent_containers/emergency_injector/random,off_y=-4)
+
+
+	long //3x1
+		xSize = 3
+		ySize = 1
+
+		// GANG_CRATE_GUN
+		flaregun
+			tier = GANG_CRATE_GUN
+			spawn_loot(var/C,var/I)
+				spawn_item(C,I,/obj/item/ammo/bullets/flare,off_x=4,scale_x=0.8,scale_y=0.8)
+				spawn_item(C,I,/obj/item/gun/kinetic/flaregun,off_x=-8,scale_x=0.75,scale_x=0.75)
+		striker
+			tier = GANG_CRATE_GUN
+			spawn_loot(var/C,var/I)
+				spawn_item(C,I,/obj/item/gun/kinetic/striker,off_x=-8,scale_x=0.75,scale_x=0.75)
+		gl
+			tier = GANG_CRATE_GUN
+			spawn_loot(var/C,var/I)
+				spawn_item(C,I,/obj/item/gun/kinetic/riot40mm,scale_x=0.8,scale_y=0.8)
+				spawn_item(C,I,/obj/item/chem_grenade/flashbang,off_x=5,off_y=-4,rot=90,scale_x=0.8,scale_y=0.8)
+				spawn_item(C,I,/obj/item/chem_grenade/flashbang,off_x=8,off_y=-4,rot=90,scale_x=0.8,scale_y=0.8)
+		dsabre
+			tier = GANG_CRATE_GUN
+			spawn_loot(var/C,var/I)
+				spawn_item(C,I,/obj/item/sword/discount,rot=45,scale_x=0.9,scale_y=0.9)
+
+		// GANG_CRATE_GEAR
+		glasses
+			tier = GANG_CRATE_GEAR
+			spawn_loot(var/C,var/I)
+				spawn_item(C,I,/obj/item/clothing/glasses/sunglasses,off_y=-2)
+				spawn_item(C,I,/obj/item/clothing/glasses/sunglasses,off_y=2)
+
+		quickhacks
+			tier = GANG_CRATE_GEAR
+			spawn_loot(var/C,var/I)
+				spawn_item(C,I,/obj/item/tool/quickhack,rot=90)
+
+
+
+		//GIMMICKS
+		money_big
+			spawn_loot(var/C,var/I)
+				spawn_item(C,I,/obj/item/currency/spacecash/thousand,off_x=4,off_y=2,scale_x=0.825,scale_y=0.825,layer_offset=0.5)
```

Comment:
I thought these pixel offsets were automated by the `loot_grid` stuff?

Reply:
these are further offsets - loot_grid will handle putting your item on the top left of the crate. off_x will do minute adjustments (like stacking 3 cash objects)